uber/models.py

```diff
@@ -379,6 +379,82 @@ def apply(self, params, *, bools=(), checkgroups=(), restricted=True, ignore_csr
             if not ignore_csrf:
                 check_csrf(params.get('csrf_token'))
 
+    def timespan(self, minute_increment=60):
```

Comment:
Yea, actually on that note, I have two thoughts on that:

1) I would like to see these functions if possible not actually render any HTML or anything presentation-specific, but just become generic utility methods (unrelated to the presentation) which the presentation layer happens to use.  timespan() is a good example.

2) could we include these presentation methods in some kind of model mixin class or something similar so they don't have to live in models.py directly?  perhaps literally using: 
```@Session.model_mixin
class Attendee:
    def html_checkgroup(........):
       # stuff
```

3) Jinja makes it hyper-convenient to call these methods directly on the model passed into the template. If these methods aren't part of the class itself, then because Jinja doesn't pass strings in for us to resolve later, it's much more clunky to build the methods.

For instance, previously in django we could write:
```{% checkgroup attendee.requested_depts %}```

What we'd get passed is the string 'attendee.requested_depts', and then from there we could resolve to 'attendee' and 'requested_depts' and get the variable value from the django context.

I guess we could still do the same thing in jinja like this:
```{{ checkgroup(attendee, 'requested_depts') }}```

instead of what we're currently doing which is:
```{{ attendee.html_checkgroup('requested_depts') }}```

.......... yea actually OK that is pretty easy to implement, probably should just change it to be that way, it'll still look like the old way too.